GEOSaana_GridComp/GSI_GridComp/etc/gmao_global_convinfo.txt

```diff
@@ -173,28 +173,28 @@
  uv       258  783    1     3.0      0      0      0   2.5  20.1   1.4   2.5  0.005500     0    0.     0.      0    0.     0.
- uv       258  784    1     3.0      0      0      0   2.5  20.1   1.4   2.5  0.005500     0    0.     0.      0    0.     0.
+ uv       258  784    1     3.0      0      0      0   2.5  20.1   1.4   2.5  0.005500     1  125.     0.      0    0.     0.
  uv       259    0   -1     3.0      0      0      0   2.5  20.1   1.4   2.5  0.005500     0    0.     0.      0    0.     0.
  uv       259  783    1     3.0      0      0      0   2.5  20.1   1.4   2.5  0.005500     0    0.     0.      0    0.     0.
- uv       259  784    1     3.0      0      0      0   2.5  20.1   1.4   2.5  0.005500     0    0.     0.      0    0.     0.
+ uv       259  784    1     3.0      0      0      0   2.5  20.1   1.4   2.5  0.005500     1  100.     0.      0    0.     0.
  uv       260    0   -1     3.0      0      0      0   2.5  20.1   1.4   2.5  0.000500     0    0.     0.      0    0.     0.
  uv       260  224   -1     3.0      0      0      0   2.5  20.1   1.4   2.5  0.000500     0    0.     0.      0    0.     0.
  uv       260  225   -1     3.0      0      0      0   2.5  20.1   1.4   2.5  0.000500     0    0.     0.      0    0.     0.
  uv       270    0   -1     3.0      0      0      0   2.5  20.1   1.4   2.5  0.000500     0    0.     0.      0    0.     0.
- uv       280    0    1     3.0      0      0      0   6.0   6.1   1.4   6.0  0.005500     0    0.     0.      0    0.     0.
- uv       280    1    1     3.0      0      0      0   6.0   6.1   1.4   6.0  0.005500     0    0.     0.      0    0.     0.
+ uv       280    0    1     3.0      0      0      0   6.0   6.1   1.4   6.0  0.005500     1   50.     0.      0    0.     0.
+ uv       280    1    1     3.0      0      0      0   6.0   6.1   1.4   6.0  0.005500     1   50.     0.      0    0.     0.
  uv       281    0   -1     3.0      0      0      0   6.0   6.1   1.4   6.0  0.005500     0    0.     0.      0    0.     0.
  uv       282    0    1     3.0      0      0      0   6.0   6.1   1.4   6.0  0.005500     0    0.     0.      0    0.     0.
  uv       284    0   -1     3.0      0      0      0   6.0   6.1   1.4   6.0  0.005500     0    0.     0.      0    0.     0.
  uv       285    0    1     3.0      0      0      0   1.4   6.1   1.4   5.0  0.000500     1  100.  1200.      0    0.     0.
  uv       286    0   -1     3.0      0      0      0   6.0   6.1   1.4   6.0  0.005500     0    0.     0.      0    0.     0.
  uv       287    0   -1     3.0      0      0      0   6.0   6.1   1.4   6.0  0.000500     0    0.     0.      0    0.     0.
  uv       289    0    1     3.0      0      0      0   5.0   6.1   1.4   5.0  0.000500     0    0.     0.      0    0.     0.
- uv       290    4   -1     3.0      0      0      0   5.0   6.1   1.4   5.0  0.000500     1  100.  1200.      0    0.     0.
```

Comment:
Since this type of ASCAT wind was not actually tested, iuse=-1 here, in the x0046wd experiment,  I'd suggest we keep the original thinning grid size which is 100 km.